public/app/types/events.ts

```diff
@@ -201,3 +201,6 @@ export class AnnotationQueryFinished extends BusEventWithPayload<AnnotationQuery
 export class TimeRangeUpdatedEvent extends BusEventWithPayload<TimeRange> {
   static type = 'time-range-updated';
 }
+export class PanelEditFinishedEvent extends BusEventWithPayload<number> {
```

Comment:
When is this event published?

Reply:
Nvm, just saw it. Would it be better to name it something like `PanelEditExitedEvent` ?

Reply:
:+1: replaced